### ISSUE TITLE
Leaky ReLU: Different defaults for alpha between darknet and TF

### ISSUE THREAD

**Issue by sjain-stanford** [jinyu121/DW2TF#2]
Defaults for leaky ReLU `alpha`:

- [Darknet](https://github.com/pjreddie/darknet/blob/680d3bde1924c8ee2d1c1dea54d3e56a05ca9a26/src/activations.h#L39) uses 0.1
- [TensorFlow](https://www.tensorflow.org/api_docs/python/tf/nn/leaky_relu) uses 0.2

DW2TF code at the moment doesn't specify alpha, hence they'd mismatch.

PR on the way for this fix and few other enhancements.

**Comment by sjain-stanford** [comment]
Same goes for BatchNorm's epsilon parameter.

- Darknet uses 1e-5
- TensorFlow uses 1e-3 by default
